python/src/cloudstorage/storage_api.py

```diff
@@ -169,6 +169,42 @@ def get_bucket_async(self, path, **kwds):
     """GET a bucket."""
     return self.do_request_async(self.api_url + path, 'GET', **kwds)
 
+  # pylint: disable=too-many-locals
+  def compose_object(self, file_list, destination_file, content_type):
+    """COMPOSE multiple objects together.
+
+    Taking a list of files it calls the put object with the compose flag
+      this call merges all the files into the destination file
+    Currently stubbed because the dev server cloudstorage_stub.py
```

Comment:
Indent (and change case). or period sentence before (I find it strange to start a sentence with "Currently")...
Also, is that sentence relevant to this file (and method)?